task/aws/resources/data_source_image.go

```diff
@@ -34,6 +34,7 @@ func (i *Image) Read(ctx context.Context) error {
 	image := i.Identifier
 	images := map[string]string{
 		"ubuntu": "ubuntu@099720109477:x86_64:*ubuntu/images/hvm-ssd/ubuntu-focal-20.04*",
+		"nvidia": "ubuntu@679593333241:x86_64:NVIDIA Deep Learning  AMI v21.02.2-*",
```

Comment:
nothing relevant but you have an extra space without the OCD that you don't have ringing?

Reply:
💯 You have eagle eyes! It turns out that the image name **has** two spaces in it.

Reply:
When I wrote this pull request, I took the effort of triple–checking it with `aws ec2 describe-images` to make sure it wasn't a typo.

Reply:
```bash
aws ec2 describe-images --region us-east-1 --filter "Name=name,Values=NVIDIA Deep Learning  AMI v21.02.2-*"
```